src/pybind/mgr/mgr_module.py

```diff
@@ -2351,7 +2351,8 @@ def is_authorized(self, arguments: Dict[str, str]) -> bool:
 
     @API.expose
     def send_rgwadmin_command(self, args: List[str],
-                              stdout_as_json: bool = True) -> Tuple[int, Union[str, dict], str]:
+                              stdout_as_json: bool = True,
+                              timeout: Optional[int] = 20) -> Tuple[int, Union[str, dict], str]:
```

Comment:
do we need to increase the default? whatever commands are affected that can be put to 30 for now?